e2e/playwright/mock_oauth_server/index.ts

```diff
@@ -24,5 +27,6 @@ const app = makeOAuthServer({
 
 const port = process.env.OAUTH_SERVER_PORT || 8080;
 app.listen(port, () => {
+    // eslint-disable-next-line no-console
```

Comment:
We can probably just get rid of this file. It's used to stand up the oauth server separate from the tests, but it's not ever used like that